app/helpers/load_resources_helper.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
```

Comment:
Note for reviewers: this file isn't tested using helper tests. Given that it's testing params and using instance variables, I think it's easier to test using request tests. Feel free to disagree!